src/main/java/io/github/thebusybiscuit/exoticgarden/Berry.java

```diff
@@ -30,8 +30,6 @@ public Berry(ItemStack item, String id, PlantType type, String texture) {
      * Returns the identifier of this Berry.
      *
      * @return the identifier of this Berry
-     *
-     * @since 1.7.0, rename of {@link #getName()}.
```

Comment:
I don't really understand why you removed that.

Reply:
still this ^